File: c++/bt.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int a, s=0; 
    cout<<"Nhap so tien gui: ";
    cin>>a;
    for(int i = 0; i<=9 ; i++){
        a = a + a*0.051;
    }
        cout<<a;
    for(int j = 0;j < 10000; j++)
    {
        a = a + a*0.051;
        s = s+1;
        if(a = 50000000)
        cout<<s;
        break;
    }
}

File: python/bt.py
a  = int(input('so tien gui = '))
s = 0
for i in range(10):
    a = a+a*0.051
print('So tien ban co sau 10 nam:',int(a))

for j in range(1000):
    a = a + a*0.051
    s = s +1
    if a == 50000000:
        print(s)